tests/integration/data_sources_and_expectations/expectations/test_expect_column_values_to_be_between.py

```diff
@@ -151,6 +157,51 @@ def test_failure(
     assert not result.success
 
 
+@pytest.mark.parametrize(
+    "min_value,max_value,expected_message",
+    [
+        pytest.param(
+            "",
+            1,
+            "values cannot be empty strings",
+            id="min_value_is_empty_string",
+        ),
+        pytest.param(
+            0,
+            "",
+            "values cannot be empty strings",
+            id="max_value_is_empty_string",
+        ),
+        pytest.param(
+            None,
+            None,
+            "min_value and max_value cannot both be None",
+            id="both_values_are_none",
+        ),
+        pytest.param(
+            "",
+            "",
+            "values cannot be empty strings",
```

Comment:
In general, my preference is to test that an error is raised, but not the specific message. If somebody changes a character in the message they will need to update the test, which in this repo, can cost quite a bit of time due to CI runtimes. Sometimes I do something like `"empty string" in actual_message` if I want to test the most important part of the message.